# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/canvas/WebGLTexture.h
# Commit: 7e2a2b1b416f
# Full Hash: 7e2a2b1b416f1a64c6548aab29e92acc9554ae7d
# Author: Jeff Gilbert <jgilbert@mozilla.com>
# Date: 2020-01-09 03:40:18
# Regressor Bug: 1477756
# File Overlap Count: 1
# Description:
#   Bug 1477756 - Fix all webgl regression tests according to CI. r=handyman
#   
#   (This is a combination of 31 commits)
#   
#   * Fix Linux compilation.
# ==============================================================================

diff -r ccfa767dba64 -r 7e2a2b1b416f dom/canvas/WebGLTexture.h
--- a/dom/canvas/WebGLTexture.h	Wed Jan 08 22:19:16 2020 +0000
+++ b/dom/canvas/WebGLTexture.h	Wed Jan 08 22:19:23 2020 +0000
@@ -211,7 +211,7 @@
                           GLenum formatEnum, const uvec3& offset,
                           const uvec3& size, const Range<const uint8_t>& src,
                           const uint32_t pboImageSize,
-                          const Maybe<uint64_t> pboOffset);
+                          const Maybe<uint64_t>& pboOffset);
 
   // CopyTexSubImage iff `!respecFormat`
   void CopyTexImage(GLenum imageTarget, uint32_t level, GLenum respecFormat,